bot/seasons/evergreen/nationday.py

```diff
@@ -0,0 +1,166 @@
+            not_available_msg = f"This country is not currently available."
+            logger.warning(not_available_msg)
+            return not_available_msg, WHITE_FLAG
+
+        async with self.http_session.get(URL+iso_code) as resp:
+            info = await resp.json()
+
+        # check if 404 error
+        if info.get('status') == 404:
+            not_found_msg = f"Could not find information on this country."
+            logger.warning(not_found_msg)
+            return not_found_msg, WHITE_FLAG
+
+        country_info = ""
+        country_info += f"**Name**: {info['name']}\n"
+        country_info += f"**Region**: {info['region']}\n"
+        country_info += f"**Capital**: {info['capital']}\n"
+        country_info += f"**Population**: {info['population']:,}\n"
+        country_info += f"**Currency**: {info['currencies'][0]['name']} ({info['currencies'][0]['symbol']})\n"
+        flag = flag_url.format(country_code=iso_code)
+
+        return country_info, flag
+
+    @command(name='nationday')
+    async def nationday(self, ctx: Context, *, arg: str) -> None:
+        """
+        \U0001F30F NationDay Help.
+
+        Enter a country name to get independence day of that country along with some basic information on the country.
+        Enter "today" to get all countries whose independence day is the current day, along with information.
+        Usage:
+        -> .nationday today
+        -> .nationday [country] (use "" for countries with space in the name)
+        Examples:
+        -> .nationday today
+        -> .nationday india
+        -> .nationday United Arab Emirates
+        -> .nationday Ivory Coast
+        """
+        arg = arg.lower()
+
+        if arg == 'today':
+            pages = await self.country_today()
+            embed = discord.Embed(
+                title='Countries that have their independence days today'
+            ).set_footer(text='Powered by the RESTCountries API.')
+            await ImagePaginator.paginate(pages, ctx, embed)
+
+        # Check if country is present
+        elif arg in self.iso_codes.keys():
+            page, date = await self.get_specific_country(arg)
+            embed = discord.Embed(
+                title=f'{arg.title()} -> {date.capitalize()}',
+                description=page[0]
+            ).set_footer(text='Powered by the RESTCountries API.')
+            embed.set_image(url=page[1])
+            await ctx.channel.send(embed=embed)
+
+        else:
+            await ctx.channel.send(
+                (
+                    "Give appropriate country name Eg. 'United States of America'\n"
+                    "**Country entered may not be available** OR an invalid option was used.\n"
+                    "Check out the help section below."
+                )
+            )
+            await ctx.send_help('nationday')
+
+
+def setup(bot: Bot) -> None:
+    """Load NationDay Cog."""
+    bot.add_cog(NationDay(bot))
+    logger.debug("NationDay cog loaded.")
```

Comment:
This is not necessary anymore, as bot will automatically send this debug.

Reply:
@ks129 The last time I worked on this, the directory structure for seasonalbot/ was different. So the fork I have has the older structure. What should I do in this case? Close this PR, make a new fork and make the changes there?
Thanks in advance!
P.S. I'm still a beginner at Git.

Reply:
I made the changes you mentioned, but I'm having a lot of issues with getting it to run. I'm using an Anaconda env and despite installing `arrow`, I keep getting a `ModuleNotFoundError` when I do `pipenv run start`. It used to work completely fine the last time I'd made a commit.
However, I was on Ubuntu 19.04 back then. Current is 20.04. Also, haven't used Ubuntu for the better part of 5 months now on my dual-booted laptop. Any suggestions as to what could be wrong?

Reply:
Try doing `pipenv sync --dev` and then `pipenv run start`?